src/tests/test_pyodide_http.py

```diff
@@ -58,3 +58,15 @@ async def test_pyfetch_unpack_archive():
         "src",
         "tests",
     ]
+
+
+@run_in_pyodide
+async def test_pyfetch_headers():
+    from pyodide.http import pyfetch
+
+    response = await pyfetch(
+        "http://files.pythonhosted.org/packages/93/8d/e1e98360dc899e533cb3dd857494f2571b129bdffcee76365009b2bb507c/example_pypi_package-0.1.0.tar.gz"
```

Comment:
Could you please rather mock this with the `httpserver` fixture as done in test_open_url to avoid HTTP calls to external servers?

Reply:
Sorry, I forgot about this PR, but I just noticed it had some activity again. I could add a test and use the convenience method, as mentioned in the previous comments.

Reply:
Hi @bartbroere, I needed this feature and found your PR :) I added a test and will merge after CI pass. Thanks for your work!